Commands.md

```diff
@@ -889,6 +890,15 @@ Create empty local branch `name`:
 $ git fresh-branch docs
 ```
 
+## git get
+
+Clone repository into `"$HOME/some-dir/<repository_name>"`:
```

Comment:
Better to mention that this command can clone repo to the configured path, as `git clone` already support cloning to a specific path.